apps/condo/pages/auth/change-password.tsx

```diff
@@ -34,15 +35,20 @@ const ChangePasswordPage: AuthPage = () => {
     const PasswordIsTooShortMsg = intl.formatMessage({ id: 'pages.auth.PasswordIsTooShort' })
     const PleaseConfirmYourPasswordMsg = intl.formatMessage({ id: 'pages.auth.PleaseConfirmYourPassword' })
     const TwoPasswordDontMatchMsg = intl.formatMessage({ id: 'pages.auth.TwoPasswordDontMatch' })
+    const ChangePasswordTokenErrorLabel = intl.formatMessage({ id: 'pages.auth.ChangePasswordTokenErrorLabel' })
+    const ChangePasswordTokenErrorMessage = intl.formatMessage({ id: 'pages.auth.ChangePasswordTokenErrorMessage' })
+    const ChangePasswordTokenErrorConfirmLabel = intl.formatMessage({ id: 'pages.auth.ChangePasswordTokenErrorConfirmLabel' })
+
     const ErrorToFormFieldMsgMapping = {}
 
     const userId = get(auth, ['user', 'id'])
 
-    const onFinish = values => {
+    const onFinish = (values: typeof initialValues) => {
```

Comment:
Can you log the user in after changing the password? If you use context from AuthLayoutContext - you will have access to sign-in action.

Reply:
do you really know a username? (identity for login)

Reply:
Yes,  as he requested token on this page - to be sure it's not expired. This token can contain email. And knowing email and a new password will be enough to sign in user. We also have a label that somehow promises the user to sign in him after changing the password. 
![image](https://user-images.githubusercontent.com/1640424/123212787-a91cd780-d4de-11eb-9e63-163b7cc8b6d9.png)

Reply:
@pahaz What is the final decision about this functionality? Does it need to be fixed?

Reply:
It's up to you. Is it have any security issues? If catch or parse a forgot password URL from browser history do you have a chance to know the identity?